content/guides/contributing-open-source-proj.md

```diff
@@ -0,0 +1,87 @@
+
+### How to Fork a Repository:
+
+First, navigate to the home page of the repository you wish to fork. Refer to the screenshot below:
+
+Here, you can see that I am currently within the atsign-foundation’s ‘[at_demos](https://github.com/atsign-foundation/at_demos)’ repository. 
+
+ ![fork-button](images/contributing-project/fork-button.png "Fork button")
+
+
+
+If you look at the top right corner of the repository’s page, you’ll notice the ‘Fork’ button and a number next to it. That number represents the number of times this specific repository has been forked. Once you select the Fork button, GitHub will prompt you which repository you wish to fork to. My go-to option is typically my own profile, where it will create an entire repository under your profile name so that you may make whatever modifications you wish. After you choose what location you wish to fork to, GitHub will do all of the hard work for you. After a few seconds or so (depending on your Internet connection speed), you’ll find that the entire repository that you forked, is now copied to wherever you told GitHub to fork it! Congratulations! You’ve forked a repository! 
+
+
+### Make Modifications to the Repository:
+
+Now, you can do whatever your heart so desires with the project! Depending on what the project is, you may wish to clone the repository to your local machine using https/, or simply commit changes directly on your forked repository (totally your choice/preference!).
+
+### Creating a Public Key with GitHub
+
+We recommend git cloning the repository using the SSH key option. You can set up your own public keys on GitHub quite easily. If you see below, you will see the Clone options, 'HTTPS, SSH, and GitHub CLI'. Select the 'SSH' option and this will provide you a password-protected SSH key to git clone the repository. 
+
+![GitHub-Code-Options](images/getting-started/Github-Code-Options.png "download theme options")
+
+In order to create a public key on your machine, open any terminal and type the command:
+
+```
+ssh-keygen
+```
+
+From here, it will ask you to choose a lcoation to store the key. We suggest just clicking 'enter' on whatever location is chosen for you. It will then ask for a passphrase, we STRONGLY recommend clicking 'enter' again to not input a passphrase. After doing this, it will ask you to input the passphrase again and click 'enter' without inputting anything. After having successully creating the key, enter the command:
+
+```dart
+// Depending on what type of terminal you are using
+// you will need to either use a forward or backslash
+// forward slash / for anything related to UNIX
+// backslash \ for Windows
+cat ~ \.ssh\id_rsa.pub
```

Comment:
Stray space after ~
cat will work in Git Bash on Windows, but in that case no worries about / or \
but cat won't work in an ordinary Windows CMD window